plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/QConstants.kt

```diff
@@ -7,3 +7,197 @@ object QConstants {
     const val Q_MARKETPLACE_URI = "https://aws.amazon.com/q/developer/"
     const val CODEWHISPERER_LOGIN_HELP_URI = "https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/auth-access.html"
 }
+
+object FeatureDevBundleConfig {
+    val ALLOWED_CODE_EXTENSIONS = setOf(
```

Comment:
need ack from /transform and chat that this will not affect their feature

Reply:
there is no overlap between this change and gumby's file zipping logic

Reply:
@rli can you share why you think this will affect /transform and chat?

Reply:
in `CodeScanSessionConfig.kt`, it is re-using `ignoreFile` func rather than `zipFiles`, so I don't think this change will affect Gumby's functionality